cli/vm/vm.go

```diff
@@ -11,6 +13,17 @@ func NewCommands() []cli.Command {
 		Name:   "vm",
 		Usage:  "start the virtual machine",
 		Action: startVMPrompt,
+		Subcommands: []cli.Command{
+			{
+				Name:  "parse",
```

Comment:
`neo-go vm parse`, but what's VMish about this function? How about `neo-go util convert`?

Reply:
Fixed.